editor/store/test/selectors.js

```diff
@@ -26,9 +26,7 @@ const {
 	getCurrentPostLastRevisionId,
 	getCurrentPostRevisionsCount,
 	getCurrentPostType,
```

Comment:
Should we also remove this one in the follow-up PR?

Reply:
`select( 'core/editor', 'getEditedPostAttribute', 'type' )` - we use that already in this PR.